bin/ci

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -eox pipefail
+
+echo "--- Installing dependencies"
+curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.16.0/minikube-linux-amd64
+chmod +x minikube
+sudo mv minikube /usr/local/bin/
+
+# minikube requires kubectl
+curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.5.2/bin/linux/amd64/kubectl
+chmod +x kubectl
+sudo mv kubectl /usr/local/bin/
+
+# KVM support for minikube
+curl -Lo docker-machine-driver-kvm https://github.com/dhiltgen/docker-machine-kvm/releases/download/v0.7.0/docker-machine-driver-kvm
+chmod +x docker-machine-driver-kvm
+sudo mv docker-machine-driver-kvm /usr/local/bin/
+
+minikube config set vm-driver kvm
+
+gem install bundler --no-ri --no-rdoc
+bundle install --jobs 4
+
+echo "--- Starting minikube"
+minikube start
+minikube status
```

Comment:
Can we wait for this to be "Running" before trying to start the tests? Looks like it was immediately in your test runs, but perhaps that wouldn't always be the case.